meta/applications/yc_s15.md

```diff
@@ -0,0 +1,402 @@
+# YC Application
+
+hackEDU's nonprofit YC application for S15.
+
+### Application
+
+**Organization name**
+
+hackEDU
+
+**Organization url, if any**
+
+https://hackedu.us
+
+**If you have an online demo, what's the url?**
+
+N/A
+
+**What will your organization do?**
+
+hackEDU is an international community of programming clubs. We think it's silly
+that students who love music can join band, students who love sports can join
+an athletic team, but students who want to learn to code have to go home and
+code alone. Our vision is that every high school in the country will offer
```

Comment:
This limits our scope to people who already know they want to code. That's not what we're about.